README.md

```diff
@@ -1,89 +1,88 @@
-## Türkçe Kaynaklar Listesi [![Build Status](https://img.shields.io/travis/fatihacet/turkcekaynaklar-com.svg?style=flat-square)](https://travis-ci.org/fatihacet/turkcekaynaklar-com)
+## Türkçe Kaynaklar Listesi
 
 Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
 
-### [https://turkcekaynaklar.com](https://turkcekaynaklar.com)
 ### [KATKI ÇAĞRISI](/Katki-Cagrisi.md)
 
 ---
 
-
-### Konu başlıkları
-- [Android](#android)
-- [Angular](#angular)
-- [Arduino](#arduino)
-- [ASP.NET](#aspnet)
-- [Bash](#bash)
-- [Bilgisayar Bilimleri](#bilgisayar-bilimleri)
-- [Blockchain](#blockchain)
-- [Bot Framework](#bot-framework)
-- [CodeIgniter](#codeigniter)
-- [C](#c)
-- [C#](#csharp)
-- [Clojure](#clojure)
-- [CodeIgniter](#codeigniter)
-- [Cpp](#cpp)
-- [Crystal](#crystal)
-- [CSS](#css)
-- [D](#d)
-- [Dart](#dart)
-- [Deep Learning](#deep-learning)
-- [Design Patterns](#design-patterns)
-- [Django](#django)
-- [Docker](#docker)
-- [Electron](#electron)
-- [Flutter](#flutter)
-- [Flux](#flux)
-- [Git](#git)
-- [GNU/Linux](#gnulinux)
-- [Go](#go)
-- [Gömülü Sistemler](#gömülü-sistemler)
-- [GraphQL](#graphql)
-- [Gulp](#gulp)
-- [Haskell](#haskell)
-- [HTML](#html)
-- [Ionic](#ionic)
-- [Java](#java)
-- [JavaScript](#javascript)
-- [jQuery](#jquery)
-- [Kotlin](#kotlin)
-- [Laravel](#laravel)
-- [Machine Learning](#machine-learning)
-- [MATLAB](#matlab)
-- [MongoDB](#mongodb)
-- [Nginx](#nginx)
-- [Node.js](#nodejs)
-- [OpenSSL](#openssl)
-- [Packer](#packer)
-- [PHP](#php)
-- [Postgres](#postgres)
-- [PowerShell](#powershell)
-- [Python](#python)
-- [Rails](#rails)
-- [React](#react)
-- [React Native](#react-native)
-- [Ruby](#ruby)
-- [Rust](#rust)
-- [RxJs](#rxjs)
-- [Sketch](#sketch)
-- [Solidity](#solidity)
-- [Sourcemod](#sourcemod)
-- [Spring Boot](#spring-boot)
-- [SQL](#sql)
-- [Swift](#swift)
-- [Titanium Mobile](#titanium-mobile)
-- [TypeScript](#typescript)
-- [Vault](#vault)
-- [Vim](#vim)
-- [Vue.js](#vuejs)
-- [WordPress](#wordpress)
-- [Xamarin](#xamarin)
-- Diğer Kaynaklar
+### Konu Başlıkları
+  - [Android](#android)
+  - [Angular](#angular)
+  - [ASP.NET](#aspnet)
+  - [Arduino](#arduino)
+  - [Bash](#bash)
+  - [Bilgisayar Bilimleri](#bilgisayar-bilimleri)
+  - [Blockchain](#blockchain)
+  - [Bot Framework](#bot-framework)
+  - [C](#c)
+  - [Csharp](#csharp)
+  - [Clojure](#clojure)
+  - [CodeIgniter](#codeigniter)
+  - [Cpp](#cpp)
+  - [Crystal](#crystal)
+  - [CSS](#css)
+  - [D](#d)
+  - [Dart](#dart)
+  - [Deep Learning](#deep-learning)
+  - [Design Patterns](#design-patterns)
+  - [Django](#django)
+  - [Docker](#docker)
+  - [Electron](#electron)
+  - [Flux](#flux)
+  - [Flutter](#flutter)
```

Comment:
T before X?

Reply:
The "Markdown All in One" plugin caused this. I'm fixing it.

Reply:
It is also wrong during the main content. I'm fixing it.